lighthouse-core/lib/page-functions.js

```diff
@@ -127,13 +123,16 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
   // TODO(paulirish): Don't clean title attribute from all elements if it's unnecessary
   const autoFillIgnoreAttrs = ['autofill-information', 'autofill-prediction', 'title'];
 
-  try {
-    // ShadowRoots are sometimes passed in; use their hosts' outerHTML.
-    if (element instanceof ShadowRoot) {
-      element = element.host;
-    }
+  // ShadowRoots are sometimes passed in; use their hosts' outerHTML.
+  if (element instanceof ShadowRoot) {
+    element = element.host;
+  }
```

Comment:
the catch block below relies on `element` not being a `ShadowRoot` anymore, so this needs to be outside of the try block. From [the PR that added it](https://github.com/GoogleChrome/lighthouse/pull/9079), it seems like the concern was the `clone` throwing, not this step anyways